examples/nirf/rfsa-rfsg-NR-analysis-only-modacc.py

```diff
@@ -0,0 +1,464 @@
+r"""Demostration on how to use RFSA, RFSG and RFmx in Analysis Only mode.

```

Comment:
Most of the examples folder names match the metadata source names. That's how they look up "release readiness" by looking in the corresponding metadata source folder for a `config.py` file. The error  `'No module config.py metadata for module_name: /home/runner/work/grpc-device/grpc-device/examples/nirf'` you're getting is because there's no corresponding metadata source folder since this isn't one-to-one with a driver/service. We'll just need to nudge this one into the include or exclude camp in stage_client_files.py.

To include this "nirf" example in the artifacts (what's bundled in installer/releases) you'd update [here](https://github.com/ni/grpc-device/blob/ad35fedd1975a3efbd84ffda3d3f2934377f0e9a/source/codegen/stage_client_files.py#L44) to something like:
```python
        if module_name in ["session", "nirf"]:
            return True
```

To exclude it we'd update [here](https://github.com/ni/grpc-device/blob/ad35fedd1975a3efbd84ffda3d3f2934377f0e9a/source/codegen/stage_client_files.py#L41) to be:
```python
if module_name in ["fake", "nirf"]:
            return False
```

Reply:
thank you for that.
Should I make that change then into stage_client_files.py and add it with the same changelist?

Reply:
Done